app/controllers/AuditController.scala

```diff
@@ -43,259 +43,287 @@ class AuditController @Inject() (implicit val env: Environment[User, SessionAuth
   }
 
   /**
-    * Returns an explore page.
+    * Returns /explore wrapper page.
     */
-  def explore(newRegion: Boolean, retakeTutorial: Option[Boolean], routeId: Option[Int], resumeRoute: Boolean) = UserAwareAction.async { implicit request =>
+  def exploreWrapper = UserAwareAction.async { implicit request =>
     val timestamp: Timestamp = new Timestamp(Instant.now.toEpochMilli)
     val ipAddress: String = request.remoteAddress
-    val qString = request.queryString.map { case (k, v) => k.mkString -> v.mkString }
-
-    val studyGroupInput: Option[String] = qString.get("studyGroup")
-    val studyGroup: String = studyGroupInput.map(g => if (g == "1" || g == "2") g else "").getOrElse("")
-
-    val retakingTutorial: Boolean = retakeTutorial.isDefined && retakeTutorial.get
 
     request.identity match {
       case Some(user) =>
-        // If the user is a Turker and has audited less than 50 meters in the current region, then delete the current region.
-        val currentMeters: Option[Float] = MissionTable.getMetersAuditedInCurrentMission(user.userId)
-        if (user.role.getOrElse("") == "Turker" &&  currentMeters.isDefined && currentMeters.get < 50) {
-          UserCurrentRegionTable.delete(user.userId)
-        }
+        WebpageActivityTable.save(WebpageActivity(0, user.userId.toString, ipAddress, "Visit_Explore_Wrapper", timestamp))
```

Comment:
Although helpful for testing, I'm guessing that we won't need to log "Visit_Explore_Wrapper" long term. Can you think of reasons that we might?

Reply:
Yeah, I just did that to match with the rest of the page navigations. I'll remove it.